web/app/components/notification.hbs

```diff
@@ -1,14 +1,31 @@
 <div class="notifications-container">
   {{#each this.flashMessages.queue as |flash|}}
-    <FlashMessage @flash={{flash}} class="notification" as |component flash|>
+    <FlashMessage
+      data-test-flash-notification
+      @flash={{flash}}
+      class="notification"
+      as |component flash close|
+    >
       <Hds::Toast
         @color={{flash.type}}
         @icon={{flash.icon}}
-        @onDismiss={{this.dismiss}}
+        @onDismiss={{close}}
```

Comment:
We used to pass in a no-op based on the now-outdated idea that `dismissOnClick` is always true.